src/rules/enums_sorted_alphabetically.js

```diff
@@ -0,0 +1,26 @@
+import { GraphQLError } from "graphql/error";
+
+export function EnumsSortedAlphabetically(context) {
+  return {
+    EnumTypeDefinition: function EnumTypeDefinition(node, key, parent, path, ancestors) {
+      var enumValues = node.values.map(val => {
+        return val.name.value;
+      });
+
+      if (!arraysEqual(enumValues, enumValues.slice().sort())) {
+        context.reportError(
+          new GraphQLError("The enum `" + node.name.value + "` should be sorted alphabetically: " + enumValues, [
```

Comment:
I think I would omit `enumValues` from the message for two reasons:

1- `enumValues` could be a pretty big array and I think it would become a distraction
2- In the future we could have an output formatter that includes snippets of the `schema.graphql` with the error message, for example:

```
10:6 The object type `user` should start with a capital letter.

type user {
     ^^^^
```

What do you think?